dev/a11y_assessments/test/text_field_password_test.dart

```diff
@@ -50,4 +50,12 @@ void main() {
       expect(textField.decoration?.hintText, isNull);
     }
   });
+
+  testWidgets('disabled text field is disabled', (WidgetTester tester) async {
+    await pumpsUseCase(tester, TextFieldPasswordUseCase());
+
+    final Finder finder = find.byKey(const Key('disabled password'));
+    final TextField textField = tester.widget<TextField>(finder);
```

Comment:
This test looks like it would still pass without this change. Is this really testing the right stuff?

Reply:
Updated test case to check to see if Semantics wrapper exists.